ext/opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -141,6 +156,11 @@ def export(
             for key, value in record.labels:
                 series.metric.labels[key] = str(value)
 
+            if self.add_unique_identifier:
+                series.metric.labels["opentelemetry_uuid"] = uuid.uuid4().hex[
```

Comment:
Unless I'm misunderstanding, wouldn't this cause a new time series to be created for every data point? https://cloud.google.com/monitoring/api/v3/metric-model#cardinality This would cause high cardinality and the data wouldn't be very useful

> You are limited to 10 labels in a metric type, but you also need to ensure than the set of possible values for any label is constrained. A small set of discrete values (like “red,” “green,” and “blue,”) is the preferred approach. Granular values, such as timestamps, should not be used.

Reply:
Whoops, yeah the uuid is supposed to be on a per exporter basis, not per export. Fixed :)